src/test/java/edu/harvard/iq/dataverse/api/HarvestingClientsIT.java

```diff
@@ -272,18 +287,18 @@ private void harvestingClientRun(boolean allowHarvestingMissingCVV)  throws Inte
         System.out.println("Waited " + i + " seconds for the harvest to complete.");
 
         Response searchHarvestedDatasets = UtilIT.search("metadataSource:" + nickName, normalUserAPIKey);
+        searchHarvestedDatasets.then().assertThat().statusCode(OK.getStatusCode());
         searchHarvestedDatasets.prettyPrint();
-        searchHarvestedDatasets.then().assertThat()
-                                .statusCode(OK.getStatusCode())
-                                .body("data.total_count", equalTo(expectedNumberOfSetsHarvested));
+        // Get all global ids for cleanup
+        JsonPath jsonPath = searchHarvestedDatasets.getBody().jsonPath();
+        int sz = jsonPath.getInt("data.items.size()");
+        for(int idx = 0; idx < sz; idx++) {
+            globalIdList.add(jsonPath.getString("data.items["+idx+"].global_id"));
+        }
+        // verify count after collecting global ids
+        assertEquals(expectedNumberOfSetsHarvested, jsonPath.getInt("data.total_count"));
```

Comment:
@stevenwinship 
The **new** intermittent error, the one introduced in #10464 the other day, is still here - the last Jenkins build has just failed again on this line with ```expected: <8> but was: <2>```. Just a matter of calling the search engine while the new datasets may still be getting indexed. 

Not your fault, strictly speaking, but since we are already working on this test, may as well. Let me just add a second of sleep here and merge it; I'm sure that's all it needs.